cc.nift.nsm.json

```diff
@@ -0,0 +1,25 @@
+{
+    "app-id": "cc.nift.nsm",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "18.08",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "nsm",
+    "modules": [
+        {
+            "name": "nsm",
+            "buildsystem": "simple",
```

Comment:
This was never resolved.

Reply:
You mean adding in no-autogen instead of buildsystem and build commands, I'm not sure how I managed to revert back after changing it, but should be sorted now.

Reply:
> Please stop opening new PRs...
> 

I changed from org.nifty-site-manager.nsm to cc.nift.nsm, wasn't sure whether to or not. Wont be opening any more PRs.